package.json

```diff
@@ -29,7 +29,7 @@
     "activationEvents": [
         "onLanguage:lflang"
     ],
-    "main": "out/extension",
+    "main": "out/main",
```

Comment:
Why this rename?

Reply:
Is it because we output a minimized version and we don't want it to clash with extension.js?